File: op 2.5/op 2.5/TBall.cpp
#include "TBall.h"
#include <iostream>
using namespace std;

TBall::TBall(double radius) {
	this->radius = radius;
	this->type = "Ball";
}

double TBall::findVolume() {
	const double pi = 3.14;
	volume = (4. / 3.) * pi * radius * radius * radius;
	return volume;
}

double TBall::findArea() {
	const double pi = 3.14;
	area = 4. * pi * radius * radius;
	return area;
}

void TBall::infoOutput() {
	cout << "Type: " << type << "\tRadius: " << radius << "\tVolume: " << volume << "\t\tArea: " << area << endl;
}

double TBall::getRadius() {
	return radius;
}

File: op 2.5 py/op 2.5 py/TBall.py
from TBody import TBody

class TBall(TBody):

    def __init__(self, radius):
        self.radius = radius
        self.type = "Ball"

    def findVolume(self):
        self.volume = 4 / 3 * 3.14 * self.radius * self.radius

    def findArea(self):
        self.area = 4 * 3.14 * self.radius * self.radius

    def infoOutput(self):
        print("Type: {:<5} Radius: {!s:<5} Volume: {!s:<10.8} Area: {!s:<10.8}".format(self.type, self.radius, self.volume, self.area))

    def getRadius(self):
        return self.radius